src/browser/modules/EditorInfo/EditorInfo.jsx

```diff
@@ -8,15 +8,22 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { withBus } from 'react-suber'
 import DisplayNodeDetails from './DisplayNodeDetails'
+import { Drawer, DrawerHeader } from 'browser-components/drawer/index'
 
 export class EditorInfo extends Component {
   render () {
     return (
       <div>
-        <DisplayNodeDetails
-          labels={this.props.selectedItem._fields[0].labels}
-          selectedItem={this.props.selectedItem._fields[0].properties}
-        />
+        {this.props.selectedItem !== undefined ? (
+          <DisplayNodeDetails
+            labels={this.props.selectedItem._fields[0].labels}
+            selectedItem={this.props.selectedItem._fields[0].properties}
+          />
+        ) : (
+          <Drawer>
+            <DrawerHeader>Editor</DrawerHeader>
```

Comment:
Will it be redundant header?
i would like to see it in action.

Reply:
Shifted the DrawerHeader to EditorInfo